src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/DocumentSpanEntry.cs

```diff
@@ -33,29 +30,33 @@ internal partial class StreamingFindUsagesPresenter
         /// </summary>
         private class DocumentSpanEntry : AbstractDocumentSpanEntry
         {
-            private readonly DocumentHighlighting.HighlightSpanKind _spanKind;
-            private readonly ClassifiedSpansAndHighlightSpan _classifiedSpansAndHighlights;
+            private readonly HighlightSpanKind _spanKind;
+            private readonly ExcerptResult _excerptResult;
 
             public DocumentSpanEntry(
                 AbstractTableDataSourceFindUsagesContext context,
                 RoslynDefinitionBucket definitionBucket,
-                DocumentSpan documentSpan,
-                DocumentHighlighting.HighlightSpanKind spanKind,
+                HighlightSpanKind spanKind,
                 string documentName,
                 Guid projectGuid,
-                SourceText sourceText,
-                ClassifiedSpansAndHighlightSpan classifiedSpans)
-                : base(context, definitionBucket, documentSpan, documentName, projectGuid, sourceText)
+                MappedSpanResult mappedSpanResult,
+                ExcerptResult excerptResult)
+                : base(context,
+                      definitionBucket,
+                      documentName,
+                      projectGuid,
+                      GetLineContainingPosition(excerptResult.Content, excerptResult.MappedSpan.Start),
+                      mappedSpanResult)
```

Comment:
➡️ This alignment is the alignment produced by our smart indentation. The current behavior might be a bug, but it's unquestionably the outcome I would expect to see in a pull request.

Reply:
> Users must not be required to manually align things, so the default alignment produced by our formatter and indenter while users work shall always be an acceptable form in a pull request.

All this does is make the IDE more and more inconsistent.   I thought we talked previously about being consistent until the point that we can get out formatter to follow the patterns and practices followed in the rest of the IDE.

Also, i disagree that this is waht our smart indenter produces.  Our smart indenter would align the arguments with the 's' in 'base'. This is aligned with teh open paren.  So this was both manually different from normal smart indenting, and doesn't follow normal alignment.  This shoudl not go in in this form.  It just allows for the codebase to continue to deviate.

I'm amenable to the idea that *users* not need to know this.  But i'm not amenable to the idea that the team itself just adds new code that now takes a codebase we worked a *lot* on consistency for and is just moving away from that.

Reply:
I dont want to get into code style arguments. I will accept anything as long as there is a diagnostics and auto code fixer for the style. doesn't need to be in box code style analyzers, or in box code fixer. just create a thing that force it and run it automatically in every CI/PR rather than asking people to manually fix it in PR.

Reply:
the discussion on that is ongoing.  I believe it's on @sharwell to drive with the right stakeholders.  In the meantime, it would be great if code could be consistent with the patterns and practices used in the IDE up until now.  There is no real good reason to deviate, and until we can get a tool to help enforce that it makes things not stand out as being different for no good reason. Thanks!